cmd/root.go

```diff
@@ -64,6 +67,9 @@ func init() {
 		key := strings.ReplaceAll(f.Name, "-", "_")
 		cobra.CheckErr(viper.BindPFlag(key, flags.Lookup(f.Name)))
 	})
+
+	flags.BoolVar(&experimental, "experimental", false, "enable experimental features")
```

Comment:
```suggestion
	flags.BoolVar(&experimental, "experimental", false, "enable experimental features (commands that require this flag may break semver)")
```

Shall we explain semver rules for this flag or do you think it's ok to be implicit?

Reply:
I think it's OK - should be clear from the flag name